internal/http/services/owncloud/ocdav/propfind/propfind.go

```diff
@@ -1147,19 +1148,11 @@ func mdToPropResponse(ctx context.Context, pf *XML, md *provider.ResourceInfo, p
 	appendMetadataProp := func(metadata map[string]string, tagNamespace string, name string, metadataPrefix string, keys []string) {
 		content := strings.Builder{}
 		for _, key := range keys {
-			lowerCaseKey := strings.ToLower(key)
+			kebabCaseKey := strcase.ToKebab(key)
```

Comment:
The keys are hardcoded above. Why even lower or kebap case them? IMO `isVariableBitrate` should just remain camel case. Any reasons against that?

Reply:
@kobergj wanted to achieve a somewhat consistent format for keys in the propfind response. From a client perspective that is a good thing to achieve. A mix of kebab, camel and snake case in the keys is annoying as hell... 

However, my strongest need is, that I can convert the keys easily to a different case so that they match an interface of mine. Types/interfaces in web are all camelCase, so I need to be able to convert as needed.